xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-test/xwiki-platform-image-lightbox-test-docker/src/test/it/org/xwiki/image/lightbox/test/ui/LightboxIT.java

```diff
@@ -80,6 +80,7 @@ void disabledLightbox(TestUtils testUtils, TestReference testReference, TestConf
 
         // Make sure that the images are displayed.
         testUtils.getDriver().navigate().refresh();
+        lightboxPage = new LightboxPage();
```

Comment:
There two lines are repeated a lot, imo that's the sign that they should be moved to a method.

Reply:
more than method, it should be moved to a PO class.

Reply:
Another approach could be to call `waitUntilPageJSIsLoaded` in methods of `LightboxPage` that are actually requiring the javascript to be loaded to work properly.